extensions/opc/src/fetchopc.cpp

```diff
@@ -65,13 +65,20 @@ namespace processors {
       ->withDescription("Specifiec the max depth of browsing. 0 means unlimited.")
       ->withDefaultValue<uint64_t>(0)->build());
 
+  core::Property FetchOPCProcessor::Lazy(
```

Comment:
I think it would be better to have a default (default "Off") here, this way we would not break previous configurations and make it easier to configure new ones.